sql/src/test/java/org/opensearch/sql/sql/SQLServiceTest.java

```diff
@@ -83,11 +83,119 @@ public void onFailure(Exception e) {
         });
   }
 
+  @Test
+  public void canExecuteJsonFormatRequest() {
+    doAnswer(invocation -> {
+      ResponseListener<QueryResponse> listener = invocation.getArgument(1);
+      listener.onResponse(new QueryResponse(schema, Collections.emptyList(), ""));
+      return null;
+    }).when(queryService).execute(any(), any());
+
+    sqlService.execute(
+        new SQLQueryRequest(new JSONObject(), "SELECT FIELD FROM TABLE", QUERY, "json"),
+        new ResponseListener<QueryResponse>() {
+          @Override
+          public void onResponse(QueryResponse response) {
+            assertNotNull(response);
+          }
+
+          @Override
+          public void onFailure(Exception e) {
+            fail(e);
+          }
+        });
+  }
+
+  @Test
+  public void canThrowUnsupportedExceptionForAggregationJsonQuery() {
+    sqlService.execute(
+        new SQLQueryRequest(new JSONObject(), "SELECT FIELD FROM TABLE GROUP BY FIELD", QUERY, "json"),
+        new ResponseListener<QueryResponse>() {
+          @Override
+          public void onResponse(QueryResponse response) {
+            assertNotNull(response);
+          }
+
+          @Override
+          public void onFailure(Exception e) {
+            assert(e instanceof UnsupportedOperationException);
```

Comment:
Maybe
```suggestion
            assertTrue(e instanceof UnsupportedOperationException);
```

Reply:
Thanks, fixed in f6db3453e5b8aabdf10a6ba12016e311d0835c6b